### PR TITLE
passing context to services, creating constants and native error

### PR DESCRIPTION
**Pull request by henrybarreto**

### CODE REVIEW

File: cli/services/namespaces.go [hunk 1]
```diff
@@ -11,9 +11,12 @@ import (
 	"github.com/shellhub-io/shellhub/pkg/validator"
 )
 
-func (s *service) NamespaceCreate(namespace, username, tenant string) (*models.Namespace, error) {
-	ctx := context.Background()
+const (
+	MaxNumberDevicesCloud      = 3
+	MaxNumberDevicesInterprise = -1
```
Comment (otavio) [inline cli/services/namespaces.go:16]: Community?

**Review by otavio — CHANGES_REQUESTED** [review]

**Review by otavio — APPROVED** [review]

**Comment by otavio** [comment]
@henrybarreto please fix the CI error so we can merge it.

**Review by otavio — DISMISSED** [review]

**Comment by henrybarreto** [comment]
It is ready to merge

File: api/store/mongo/migrations.go [hunk 2]
```diff
@@ -17,7 +17,7 @@ import (
 func ApplyMigrations(db *mongo.Database) error {
 	logrus.Info("Creating lock for the resource migrations")
 
-	lockClient := lock.NewClient(db.Collection("locks", options.Collection().SetWriteConcern(writeconcern.New(writeconcern.WMajority()))))
+	lockClient := lock.NewClient(db.Collection("locks", options.Collection().SetWriteConcern(writeconcern.New(writeconcern.WMajority())))) //nolint:staticcheck
```
Comment (otavio) [inline api/store/mongo/migrations.go:20]: Please fix this in a specific commit. Could this be fixes in another PR as this is unrelated.

**Pull request merged**